graphene_sqlalchemy/tests/test_batching.py

```diff
@@ -82,11 +116,11 @@ async def test_many_to_one(session_factory):
     session = session_factory()
 
     reporter_1 = Reporter(
-      first_name='Reporter_1',
+        first_name='Reporter_1',
     )
     session.add(reporter_1)
     reporter_2 = Reporter(
-      first_name='Reporter_2',
+        first_name='Reporter_2',
```

Comment:
unfortunately there were some flake8 errors I had to fix, so ignore these indentation changes